crates/revm/revm-inspectors/src/tracing/types.rs

```diff
@@ -240,12 +240,12 @@ impl CallTraceNode {
 
             // If the opcode is a call, put the child trace on the stack
             match step.op.u8() {
-                opcode::CREATE |
-                opcode::CREATE2 |
-                opcode::DELEGATECALL |
-                opcode::CALL |
-                opcode::STATICCALL |
-                opcode::CALLCODE => {
+                opcode::CREATE
+                | opcode::CREATE2
+                | opcode::DELEGATECALL
+                | opcode::CALL
+                | opcode::STATICCALL
+                | opcode::CALLCODE => {
```

Comment:
I'd like to make this a helper function

Reply:
i think this comment was to be on the function `vm_trace` in `src/tracing/builder/parity`? sorry i realize my fmt touched this when it shouldnt have

if u can confirm i can fix